style.css

```diff
@@ -9,7 +11,7 @@ body {
   background-color: #e5e5e5;
 }
 
-@media all and (max-width: 1020) {
+@media all and (max-width: 768px) {
   body {
     padding: auto;
```

Comment:
- [ ] Kindly stick to the design as much as possible. for a better user experience. 
`hint: You may consider adding the font of your elements and readjust the placement of elements on your page.`

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201911093-3b822158-e53c-4845-b782-12fb2df310a1.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201911182-a8c9dad2-b72e-434a-b563-c4ea80cab728.png)